large-transfer-out-py/src/agent.py

```diff
@@ -60,7 +50,12 @@ def detect_suspicious_native_transfers(w3, transaction_event: forta_agent.transa
                 'severity': FindingSeverity.Low,
                 'type': FindingType.Info,
                 'metadata': {
-                    'anomaly_score': anomaly_score,
+                    'anomaly_score': calculate_alert_rate(
+                        CHAIN_ID,
+                        BOT_ID,
+                        'LARGE-TRANSFER-OUT',
+                        ScanCountType.TRANSFER_COUNT,
```

Comment:
Could you use `ScanCountType.LARGE_VALUE_TRANSFER_COUNT` for the denominator?

https://github.com/forta-network/bot-alert-rate/blob/main/python/bot_alert_rate/calculate_alert_rate.py#L18